src/main/webapp/about.jsp

```diff
@@ -1,334 +1,2132 @@
+                target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>Anthony Schneider</td>
+          <td>
+            <a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3ABr35Ba56"
+                target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>Anurag Sharma</td>
+          <td>
+            <a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Aanu0012"
+                target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>Aravind Putrevu</td>
+          <td>
+            <a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3AAravindPutrevu"
+                target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>Ashwin Oke</td>
+          <td>
+            <a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Aokeashwin"
+                target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>Atheesan Sornalingam</td>
+          <td>bug reporting/fixing, enhancements</td>
+        </tr>
+        <tr>
+          <td>Avijit Gupta</td>
+          <td>
+            <a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3A526avijitgupta"
+                target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>Aygiz Shaymardanov</td>
+          <td>
+            <a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Aaigyz"
+                target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>Bartłomej Weber</td>
+          <td>
+            <a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3ABedol"
+                target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>Bhargav Thanki</td>
+          <td>
+            <a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Abmthanki"
+                target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>Billy Zafack</td>
+          <td>
+            <a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3ABillyZafack"
+                target="_blank" rel="noopener noreferrer">bug reporting/fixing, enhancements
+            </a>
+      
```

Comment:
There was my mistake while merging conflicts which is resolved now.
Also already checked files manually